model/gemini.go

```diff
@@ -59,14 +60,14 @@ func (p *GeminiModelProvider) calculatePrice(modelResult *ModelResult) error {
 		modelResult.ResponseTokenCount = modelResult.TotalTokenCount
 	}
 
+	model := p.subType
 	var inputPricePerThousandTokens, outputPricePerThousandTokens float64
-	switch p.subType {
-	case "Gemini Pro":
+	switch {
+	case strings.Contains(model, "gemini-pro"):
```

Comment:
other models? only pro?

Reply:
there is only gemini pro model for now.
But there are legacy PaLM models. I'll add lines here.

Reply:
No need to add those, thanks